ui-components/src/blocks/ImageCard.tsx

```diff
@@ -26,7 +26,7 @@ const ImageCard = (props: ImageCardProps) => {
   const image = (
     <div className="image-card__wrapper">
       <figure className="image-card">
-        {props.imageUrl && <img src={props.imageUrl} alt={props.title} />}
+        {props.imageUrl && <img src={props.imageUrl} alt={"A picture of the building"} />}
```

Comment:
This comes from the rule "Ensure image alternative is not repeated as text", which is a best-practice not a WCAGAA rule, but it does seem like it would be annoying/redundant. 

More info for the rule [from Deque](https://dequeuniversity.com/rules/axe/4.2/image-redundant-alt?application=axe-puppeteer). An alternative is to leave the alt empty in this case if it's not providing value. Up to you all what you think would be a better solution here

Reply:
Would be nice to add a prop for description instead of this, since for now it's not actually a building.